src/App.jsx

```diff
@@ -11,6 +12,7 @@ return (
     <Routes>
       <Route path="/" element={<Home />} />
       <Route path="/articles/:article_id" element={<SingleArticle />} />
+      <Route path="/articles/:article_id/comments" element={<CommentsList />} />
```

Comment:
Why do you need the article comments on a separate page, if you are viewing them below the article too?

Reply:
It makes more sense to me to just view them below the article, as one their own page they are separated from the context of what the comments are talking about 😄